guides/common/assembly_configuring-puppet.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context-of-assembly_configuring-puppet: {context}]
+
+ifndef::context[]
+[id="assembly_configuring-puppet"]
+endif::[]
+ifdef::context[]
+[id="assembly_configuring-puppet_{context}"]
+endif::[]
+= Configuring Puppet
+
+:context: assembly_configuring-puppet
+
+[role="_abstract"]
+Puppet is an optional component in {ProjectX}.
```

Comment:
IMHO there are efforts to not use `{ProjectX}` any longer due to Satellite 6 also omitting the `6`.